integration-tests/pom.xml

```diff
@@ -107,6 +107,7 @@
     <dependency>
       <groupId>io.rest-assured</groupId>
       <artifactId>rest-assured</artifactId>
+      <version>4.2.0</version>
```

Comment:
Quarkus BOM seems to be importing that version already, so I'm not sure we need to explicitly declare the version here.

Reply:
Nevermind I read your explanations in the Jira ticket.